ansible/roles/kind/kind_cluster_create/templates/kind.yaml.j2

```diff
@@ -1,5 +1,12 @@
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
+{% if is_ovn_kubernetes %}
```

Comment:
why not checking the project variable here directly instead of defining the variable in the kind create role vars?

Reply:
`is_ovn_kubernetes` it is more readable and clean with boolean var